content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md

```diff
@@ -92,6 +92,36 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
 }
 ```
 
+You can also turn this behaviour off by adding the following code to your node configuration:
```

Comment:
Again, I cannot see what "this" refers to here.

Reply:
updated